### PR TITLE
[CMake][Torch] fix cmake build

### PR DESCRIPTION
**Pull request by fortianyou**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -165,5 +165,13 @@ jobs:
...
     uses: ./.github/workflows/cpu_reusable.yml
     with:
-      extra_envs: -e TORCH_BLADE_BUILD_WITH_CUDA_SUPPORT=OFF -e TORCH_BLADE_CI_BUILD_TORCH_VERSION=1.8.1+cpu
+      extra_envs: -e TORCH_BLADE_BUILD_WITH_CUDA_SUPPORT=OFF
+        -e TORCH_BLADE_CI_BUILD_TORCH_VERSION=1.8.1+cpu
+      exec_command: bash ./scripts/ci/test_pytorch_blade.sh
+  CPU-CMAKE-TORCH181:
+    uses: ./.github/workflows/cpu_reusable.yml
+    with:
+      extra_envs: -e TORCH_BLADE_BUILD_WITH_CUDA_SUPPORT=OFF
```
[the remark this answers is not in the document]
Comment (fortianyou) [reply .github/workflows/main.yml:174]: Currently, yes. But, I think we would like to maintain bazel build only in long term.

File: pytorch_blade/CMakeLists.txt [hunk 2]
```diff
@@ -145,7 +145,7 @@ resolve_env(PYTORCH_VERSION_STRING)
 resolve_env(PYTORCH_MAJOR_VERSION)
 resolve_env(PYTORCH_MINOR_VERSION)
 target_compile_definitions(torch_blade PUBLIC
-   -DPYTORCH_VERSION_STRING=${PYTORCH_VERSION_STRING}
+   -DPYTORCH_VERSION_STRING="${PYTORCH_VERSION_STRING}"  # a str
```
[the remark this answers is not in the document]
Comment (fortianyou) [reply pytorch_blade/CMakeLists.txt:148]: Actually, the comment here wants to clarify that it's a string.

**Review by Yancey1989 — APPROVED** [review]
LGTM

File: pytorch_blade/CMakeLists.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (qiuxiafei) [reply pytorch_blade/CMakeLists.txt:148]: Just interested in what if the quotes are not here ...

[the remark this answers is not in the document]
Comment (fortianyou) [reply pytorch_blade/CMakeLists.txt:148]: A compilation error will raise.

**Pull request merged**